components/multi-platform-controller/production-downstream/kflux-rhel-p01/host-config.yaml

```diff
@@ -743,50 +743,99 @@ data:
   host.s390x-large-static-2.concurrency: "4"
 
   # PPC64LE 4cores(32vCPU) / 128GiB RAM / 2TB disk
-  host.ppc64le-static-0.address: "10.130.78.70"
-  host.ppc64le-static-0.platform: "linux/ppc64le"
-  host.ppc64le-static-0.user: "root"
-  host.ppc64le-static-0.secret: "ibm-ppc64le-ssh-key"
-  host.ppc64le-static-0.concurrency: "8"
-
-  host.ppc64le-static-1.address: "10.130.78.73"
+  # host.ppc64le-static-0.address: "10.130.78.70"
+  # host.ppc64le-static-0.platform: "linux/ppc64le"
+  # host.ppc64le-static-0.user: "root"
+  # host.ppc64le-static-0.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-0.concurrency: "8"
+
+  # host.ppc64le-static-1.address: "10.130.78.73"
+  # host.ppc64le-static-1.platform: "linux/ppc64le"
+  # host.ppc64le-static-1.user: "root"
+  # host.ppc64le-static-1.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-1.concurrency: "8"
+
+  # host.ppc64le-static-2.address: "10.130.78.74"
+  # host.ppc64le-static-2.platform: "linux/ppc64le"
+  # host.ppc64le-static-2.user: "root"
+  # host.ppc64le-static-2.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-2.concurrency: "8"
+
+  # host.ppc64le-static-3.address: "10.130.78.68"
+  # host.ppc64le-static-3.platform: "linux/ppc64le"
+  # host.ppc64le-static-3.user: "root"
+  # host.ppc64le-static-3.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-3.concurrency: "8"
+
+  # host.ppc64le-static-4.address: "10.130.78.78"
+  # host.ppc64le-static-4.platform: "linux/ppc64le"
+  # host.ppc64le-static-4.user: "root"
+  # host.ppc64le-static-4.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-4.concurrency: "8"
+
+  # host.ppc64le-static-5.address: "10.130.78.76"
+  # host.ppc64le-static-5.platform: "linux/ppc64le"
+  # host.ppc64le-static-5.user: "root"
+  # host.ppc64le-static-5.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-5.concurrency: "8"
+
+  # host.ppc64le-static-6.address: "10.130.78.71"
+  # host.ppc64le-static-6.platform: "linux/ppc64le"
+  # host.ppc64le-static-6.user: "root"
+  # host.ppc64le-static-6.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-6.concurrency: "8"
+
+  # host.ppc64le-static-7.address: "10.130.78.77"
+  # host.ppc64le-static-7.platform: "linux/ppc64le"
+  # host.ppc64le-static-7.user: "root"
+  # host.ppc64le-static-7.secret: "ibm-ppc64le-ssh-key"
+  # host.ppc64le-static-7.concurrency: "8"
+
+# New Workspace Machines -- incident - itn-2025-00225
```

Comment:
instead of commenting, we could remove these entries. They'll still be present in `git`'s history.